src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let married = '';
+
+  if (person.isMarried && person.sex === 'f') {
+    married = (
+      <p className="Person__partner">{person.partnerName} is my husband</p>
+    );
+  } else if (person.isMarried && person.sex === 'm') {
+    married = (
+      <p className="Person__partner">{person.partnerName} is my wife</p>
+    );
+  } else {
+    married = <p className="Person__partner"> I am not married</p>;
+  }
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```